recipes/ignition-cmake/all/conanfile.py

```diff
@@ -30,15 +34,21 @@ def _configure_cmake(self):
         self._cmake = CMake(self)
         self._cmake.definitions["CMAKE_INSTALL_DATAROOTDIR"] = "lib"
         self._cmake.definitions["SKIP_component_name"] = False
-
         self._cmake.configure(source_folder=self._source_subfolder)
         return self._cmake
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename(glob.glob("ign-cmake*")[0], self._source_subfolder)
+        version_major = self.version.split(".")[0]
```

Comment:
```suggestion
        version_major = tools.Version(self.version).major
```
Don;t need to accept, only bikeshedding, but now you know there is an easier solution.

Reply:
your suggestion looks more conan-ish to me. thanks